client/python/docs/README.md

```diff
@@ -10,7 +10,7 @@ Usage
 Easy way:
 - Ensure all protobufs files needed for the client are generated by running
   `make python` from the repository root.
-- `tox -edocs` will create a valid virtual environment and use it to generate
+- `tox -e docs` will create a valid virtual environment and use it to generate
```

Comment:
FWIW this works with or without the space, but I'm OK adding the space

Reply:
Never new that! Good to know ;))